source/scripting_v3/GTA/PathFind.cs

```diff
@@ -292,7 +292,27 @@ public static bool GetNthClosestVehicleNodePositionWithHeading(Vector3 position,
 		public static bool RequestPathNodesInAreaThisFrame(Vector2 min, Vector2 max) => Function.Call<bool>(Hash.REQUEST_PATH_NODES_IN_AREA_THIS_FRAME, min.X, min.Y, max.X, max.Y);
 
 		/// <summary>
-		///  Switches vehicle nodes an angled area. <see cref="Vehicle"/>s will drive on to roads that are switched on and new <see cref="Vehicle"/>s will be generated on them.
+		/// Toggles ped nodes in a cubic area. <see cref="Ped"/>s will walk in areas that are switched on and new <see cref="Ped"/>s will be generated in them.
+		/// A ped node is switched off, no <see cref="Ped"/>s should be created in it and <see cref="Ped"/>s that already exist will try to avoid walking through it.
+		/// To undo effects of this method, use <see cref="SetPedNodesBackToOriginal(Vector3, Vector3, bool)"/>.
+		/// </summary>
+		/// <param name="min">The minimum position of the region.</param>
+		/// <param name="max">The maximum position of the region.</param>
+		/// <param name="active">Specifies wheter ped nodes in area should be switched on or off. If <see langword="true"/>, they will be switched on.</param>
+		/// <param name="forceAbortCurrentPath">Forces any active pathfinding requests to abort before this function executes.</param>
+		public static void SwitchPedNodesInArea(Vector3 min, Vector3 max, bool active, bool forceAbortCurrentPath) =>
+			Function.Call(Hash.SET_PED_PATHS_IN_AREA, min.X, min.Y, min.Z, max.X, max.Y, max.Z, active, forceAbortCurrentPath);
+		/// <summary>
+		/// Sets all ped nodes in area back to their original state as per area and ynd file defines (which is loaded as <c>CPathRegion</c> in the game process memory).
+		/// </summary>
+		/// <param name="min">The minimum position of the region.</param>
+		/// <param name="max">The maximum position of the region.</param>
+		/// <param name="forceAbortCurrentPath">Forces any active pathfinding requests to abort before this function executes.</param>
+		public static void SetPedNodesBackToOriginal(Vector3 min, Vector3 max, bool forceAbortCurrentPath) =>
+			Function.Call(Hash.SET_PED_PATHS_BACK_TO_ORIGINAL, min.X, min.Y, min.Z, max.X, max.Y, max.Z, forceAbortCurrentPath);
```

Comment:
They should not be `SwitchPedNodesInArea` and `SetPedNodesBackToOriginal`, because the natives won't modify path nodes that are used for vehicle nodes and that you can select with "Path" selection in CodeWalker. Let's just use `SwitchPedPathsInArea` and `SetPedPathsBackToOriginal`. The natives mutate `CNavMesh` instances stored in `aiNavMeshStore` iirc, btw. At least they access some of the same nav meshes you can select with "Nav Mesh" selection in CodeWalker.

Apologies for late responce for this, I often miss things.